pkg/doc/util/jwkkid/kid_creator.go

```diff
@@ -194,13 +196,11 @@ func BuildJWK(keyBytes []byte, kt kms.KeyType) (*jwk.JWK, error) {
 		if err != nil {
 			return nil, fmt.Errorf("buildJWK: failed to build JWK from ecdsa DER key: %w", err)
 		}
-		// TODO remove `case kms.ED25519Type` in CreateKID() and uncomment below case when go-jose fixes Ed25519
-		//      JWK thumbprint. Also remove `createED25519KID(keyBytes []byte)` function further below.
-	// case kms.ED25519Type:
-	//	j, err = jwksupport.JWKFromKey(ed25519.PublicKey(keyBytes))
-	//	if err != nil {
-	//		return nil, fmt.Errorf("buildJWK: failed to build JWK from ed25519 key: %w", err)
-	//	}
+	case kms.ED25519Type:
```

Comment:
this is good to uncomment, but go-jose needs to be updated to pick up the fixed thumbprint computation as mentioned in the previous comment.